app/src/main/java/org/p2p/wallet/moonpay/extensions/SellStubs.kt

```diff
@@ -0,0 +1,38 @@
+package org.p2p.wallet.moonpay.extensions
+
+import org.p2p.core.utils.Constants
+import org.p2p.wallet.moonpay.model.SellTransaction
+import org.p2p.wallet.moonpay.model.SellTransactionAmounts
+import org.p2p.wallet.moonpay.model.SellTransactionMetadata
+import org.p2p.wallet.moonpay.repository.sell.SellTransactionFiatCurrency
+import org.p2p.wallet.utils.toBase58Instance
+import java.math.BigDecimal
+
+object SellStubs {
```

Comment:
for easier testing in debug mode, will be removed by proguard in release and feature